lib/ratelimiter.ts

```diff
@@ -1,63 +1,34 @@
 /**
- * @class Ratelimiter
- * @module Ratelimiter
- * @classdesc Ratelimiter for Sagiri
+ *  Ratelimiter for Sagiri
```

Comment:
Won't this break documentation generation?

Reply:
We don't need publicly document the Ratelimiter as it's an internal util. It was marked as private anyway so this doesn't change anything as it wasn't generated in docs before.